src/core/core.scale.js

```diff
@@ -538,6 +542,40 @@ module.exports = Element.extend({
 		return rawValue;
 	},
 
+	_parseValue: function(raw) {
+		var value, start, end, min, max;
+
+		if (helpers.isArray(raw)) {
+			value = this.getRightValue(raw);
+			start = value[0];
+			end = value[1];
+			min = Math.min(start, end);
+			max = Math.max(start, end);
+		} else {
+			value = +this.getRightValue(raw);
+			start = undefined;
+			end = value;
+			min = value;
+			max = value;
+		}
+
+		return {
+			min: min,
+			max: max,
+			start: start,
+			end: end
+		};
+	},
+
+	getScaleLabel: function(rawValue) {
```

Comment:
Shouldn't this method be private as well? At some point we may want to have it public but for now I would keep it private in case we change this approach (@kurkle @nagix @benmccann)

```javascript
/**
 * @private
 */
_getScaleLabel: function(rawValue) {
```

Reply:
+1 to making it private

Reply:
done